src/store/project/project.ts

```diff
@@ -30,14 +30,20 @@ export class Project {
 	 */
 	@MobX.observable private pages: PageRef[] = [];
 
+	/**
+	 * Path to the preview frame, relative to the projects.yaml file.
+	 */
+	@MobX.observable private previewFrame: string;
+
 	/**
 	 * Creates a new project.
 	 * @param id The technical (internal) ID of the project.
 	 * @param name The human-friendly name of the project.
 	 */
-	public constructor(id: string, name: string) {
+	public constructor(id: string, name: string, previewFrame: string) {
```

Comment:
`previewFrame` is missing as a `@param` in the constructor description

Reply:
done